programs/associated-token-account/Create.go

```diff
@@ -191,10 +201,20 @@ func (inst *Create) EncodeToTree(parent treeout.Branches) {
 }
 
 func (inst Create) MarshalWithEncoder(encoder *bin.Encoder) error {
-	return encoder.WriteBytes([]byte{}, false)
+	if inst.Idempotent {
+		if err := encoder.Encode(inst.Idempotent); err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
 func (inst *Create) UnmarshalWithDecoder(decoder *bin.Decoder) error {
+	if decoder.HasRemaining() {
+		if err := decoder.Decode(&inst.Idempotent); err != nil {
+			return err
+		}
+	}
```

Comment:
When decoding, the byte array may have nothing in it, or it might have the value `1`